docs/mdbook/src/android/getting-started-guide.md

```diff
@@ -0,0 +1,113 @@
+# Quickstart
+
+1. Add bintray Maven repositories to your project-level Gradle file (usually `<project>/<app-module>/build.gradle`).
+
+    ```gradle
+    allprojects {
+        repositories {
+        ...
+        mavenCentral()                
+        }
+    }
+    ```
+
+2. Add the library as a dependency into your module Gradle file (usually `<project>/<app-module>/build.gradle`). Replace `<version>` with the latest MapLibre GL Native version (e.g.: `org.maplibre.gl:android-sdk:10.0.2`). Visit [https://mvnrepository.com/artifact/org.maplibre.gl/android-sdk](https://mvnrepository.com/artifact/org.maplibre.gl/android-sdk) to view the version history of MapLibre GL Native for android. 
+
+    ```gradle
+    dependencies {
+        ...
+        implementation 'org.maplibre.gl:android-sdk:<version>'
+        ...
+    }
+    ```
+
+3. Sync your Android project with Gradle files.
+
+4. Add a `MapView` to your layout XML file (usually `<project>/<app-module>/src/main/res/layout/activity_main.xml`).
+
+    ```xml
+    ...
+    <com.mapbox.mapboxsdk.maps.MapView
+        android:id="@+id/mapView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        />
+    ...
+    ```
+
+5. Initialize the `MapView` in your `MainActivity` file by following the example below:
+
+    ```kotlin
+    import androidx.appcompat.app.AppCompatActivity
+    import android.os.Bundle
+    import android.view.LayoutInflater
+    import com.mapbox.mapboxsdk.Mapbox
+    import com.mapbox.mapboxsdk.camera.CameraPosition
+    import com.mapbox.mapboxsdk.geometry.LatLng
+    import com.mapbox.mapboxsdk.maps.MapView
+
+    class MainActivity : AppCompatActivity() {
+
+        // Declare a variable for MapView
+        private lateinit var mapView: MapView
+
+        override fun onCreate(savedInstanceState: Bundle?) {
+            super.onCreate(savedInstanceState)
+            
+            // Init MapLibre
+            Mapbox.getInstance(this)
+
+            // Init layout view
+            val inflater = LayoutInflater.from(this)
+            val rootView = inflater.inflate(R.layout.activity_main, null)
+            setContentView(rootView)
+
+            // Init the MapView
+            mapView = rootView.findViewById(R.id.mapView)
+            mapView.getMapAsync { map ->
+                map.setStyle("https://demotiles.maplibre.org/style.json")
+                map.cameraPosition = CameraPosition.Builder().target(LatLng(0.0,0.0)).zoom(1.0).build()
+            }
+        }
+
+        override fun onStart() {
+            super.onStart()
+            mapView?.onStart()
```

Comment:
`mapView` cannot be `null`...

Reply:
I removed `?` from `mapView` in this commit d826413c683d9d396345171edf6e237351d211f7.